### ISSUE TITLE
Performance drop using the mod, down more than 40+ fps

### ISSUE THREAD

**Issue by Auferoz** [Lyall/FFXVIFix#81]
Hi, normally without the mod, the game runs at 140fps / 160fps and in the cinematics at 30fps, then when I add the mod that basically is to extract the .zip files in the root folder of the game, with the same graphic settings, the game goes to 80fps / 100fps. anyone know why this happens ? I just want the cinematics are not capped at 30fps, otherwise I do not care a little bit the truth.

Setup:

GPU: Nvidia Geforce 4070 Super
CPU: Intel Core i7-9700KF
RAM: 2x8 GB DDR4 2666mhz
SSD: nvme m.2 1tb samsung

**Comment by Auferoz** [comment]
[FFXVIFix.log](https://github.com/user-attachments/files/17157066/FFXVIFix.log)
Hello, I attach the file

**Comment by Suneaterchl** [comment]
smae issue as the man, i have an 4060 ti and with the fix game runs like 40-60 fps and without patch runs 80-110

**Comment by Lyall** [comment]
Based on your log it looks like everything is good. Not sure why it would be impacting performance for you. 

@Suneaterchl, you're not running an ultrawide display are you? Rendering at a proper ultrawide resolution will have a perf hit, but that's to be expected as the vanilla game will only ever display 16:9 resolutions.

**Comment by Suneaterchl** [comment]
no, i dont have an ultrawide resolution. should i put my fix resolution value to false?

**Comment by Lyall** [comment]
> no, i dont have an ultrawide resolution. should i put my fix resolution value to false?

No you don't need to, the functionality is automatic and it won't be impacting performance.

> Hi, I have been testing and when I apply the mod and start the game it gives me worse performance as mentioned above, but when I close the game and start it again it runs normally, but if I restart the computer the fps drop occurs again and I have to close and run the game again.

That's odd. It sounds like it could be be related to VRAM filling up maybe? I've seen more than a few reports of this issue where performance just tanks at a certain point and they have to restart the game. I was hoping the latest game update would help but it doesn't seem like it has for everyone. 

Overall I don't think this issue is related to FFXVIFix. I think the best hope is probably to wait for further game updates and see if that helps things.
